src/Components/Activities/index.jsx

```diff
@@ -40,16 +120,70 @@ const Activities = () => {
     setShowModal(false);
   };
 
+  const closeErrorModal = () => {
```

Comment:
You could refactor these functions that handle the state of the modal component, but this is just a suggestion :)
```suggestion
const handleCloseModal = (modalStateSetter) => {
  modalStateSetter(false);
};
```